### ISSUE TITLE
Export only desired metrics

### ISSUE THREAD

**Issue by surbhig213** [yabeda-rb/yabeda-sidekiq#22]
Hey @Envek ,

Right now hitting /metrics displays all the default metrics. How can I restrict it to display only those metrics which I need?

**Comment by Envek** [comment]
Right now it is not possible to suppress export of some arbitrary metrics. All netrics collected in code running inside Sidekiq worker process, will be collected and exposed by yabeda-prometheus on /metrics endpoint of this process.

Can you please clarify, what are these “default” metrics you are talking about?

If you don't want to collect some of your custom metrics, you can place them inside `unless ::Sidekiq.server?` block.

**Comment by surbhig213** [comment]
Hey @Envek , by default metrics I mean all the 'Local per-process metrics' and 'Global cluster-wide metrics' defined in the Readme file. My question is how can I export just a few of them and not all of them.

**Comment by Envek** [comment]
No, there is no way to do that currently. You can only disable the whole “Global cluster-wide metrics” group with `collect_cluster_metrics` setting.

But why do you need that? I still can't get the use case.
